docs/userguide.md

```diff
@@ -528,6 +530,35 @@ employed by deployment:
 NAMESPACED=y FEDERATION_NAMESPACE=<namespace> ./scripts/delete-federation.sh
 ```
 
+## Local Value Retention
+
+In most cases, the federation sync controller will overwrite any
+changes made to resources it manages in member clusters.  The
+exceptions are as follows:
+
+| Resource Type   | Fields                    | Requirement   |
+|-----------------|---------------------------|-------------------------------------------------------------------------------|
+| All             | metadata.resourceVersion  | Updates require the most recent resourceVersion for concurrency control.     |
+| Scalable        | spec.replicas             | The HPA controller may be managing the replica count of a scalable resource. |
+| Services        | spec.clusterIP,spec.ports | The services controller may be managing these fields.                        |
+| ServiceAccounts | secret                    | The serviceaccounts controller may be managing this field                    |
+
+
+### Scalable Resources
+
+For scalable resources (those that have a scale subtype
+e.g. `ReplicaSet` and `Deployment`), retention of the `spec.replicas`
+field is controlled by the `retainReplicas` boolean field of the
```

Comment:
It does not mention anywhere that except `spec.replicas` no other fields listed in the table in the previous section can be controlled by user. To avoid confusion, it might make sense to add a note.

Reply:
Done